watertap/tools/oli_api/conftest.py

```diff
@@ -88,9 +91,37 @@ def oliapi_instance(
         "config_file": cred_file_path,
     }
     credential_manager = CredentialManager(**credentials, test=True)
-    credential_manager.login()
-    with OLIApi(credential_manager, test=True) as oliapi:
+    with OLIApi(credential_manager, interactive_mode=False) as oliapi:
         oliapi.get_dbs_file_id(str(local_dbs_file))
         yield oliapi
     with contextlib.suppress(FileNotFoundError):
         cred_file_path.unlink()
+
+
+@pytest.fixture
+def flash_instance(scope="session"):
+    flash = Flash()
+    yield flash
+
+
+@pytest.fixture
+def source_water(scope="session"):
+    return {
+        "temperature": 298.15,
+        "pressure": 101325,
+        "components": {
+            "Cl_-": 870,
+            "Na_+": 739,
+            "SO4_2-": 1011,
+            "Mg_2+": 90,
+            "Ca_2+": 258,
+            "K_+": 9,
+            "HCO3_-": 385,
+            "SiO2": 30,
+        },
+        "units": {
+            "temperature": pyunits.K,
+            "pressure": pyunits.Pa,
+            "components": pyunits.mg / pyunits.L,
+        },
+    }
```

Comment:
Not sure what this is being used for.

Reply:
This input should be used to generate a test dbs file and provide state variable info for flash calculations. It's supposed to represent what would be extracted from a stateblock